upgrade/upgrade-scripts/execute

```diff
@@ -382,6 +382,17 @@ dpkg-query -Wf '${Conffiles}\n' | awk '$3 == "obsolete" {print $1}' |
 		dpkg-query -W "$package" &>/dev/null ||
 			die "package '$package' is not installed"
 
+		#
+		# If the configuration file was moved to another package it
+		# will be listed for the original package as "obsolete" while
+		# also being listed as non-obsolete for the destination package.
+		#
+		if [[ $(dpkg-query -Wf '${Conffiles}\n' "$package" |
```

Comment:
In your example, what is `package` going to be? the new or old package name? Presumably, it has to be the new package for this to work, right?

Reply:
That's right, package is going to be the new package.